core/src/test/java/com/klaytn/caver/DynamicFeeTest.java

```diff
@@ -0,0 +1,113 @@
+package com.klaytn.caver;
+
+import com.klaytn.caver.methods.response.TransactionReceipt;
+import com.klaytn.caver.transaction.TxPropertyBuilder;
+import com.klaytn.caver.transaction.type.SmartContractDeploy;
+import com.klaytn.caver.transaction.type.ValueTransfer;
+import com.klaytn.caver.utils.Utils;
+import com.klaytn.caver.wallet.keyring.AbstractKeyring;
+import com.klaytn.caver.wallet.keyring.SingleKeyring;
+import org.junit.Before;
+import org.web3j.protocol.core.BatchRequest;
+import org.web3j.utils.Numeric;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+public class DynamicFeeTest {
+    static Caver caver;
+    static SingleKeyring rich, sender, feePayer;
+
+    @Before
+    public void setUp() throws Exception {
+        caver = new Caver(Caver.DEFAULT_URL);
+        rich = caver.wallet.keyring.createFromPrivateKey("0x871ccee7755bb4247e783110cafa6437f9f593a1eaeebe0efcc1b0852282c3e5");
+        caver.wallet.add(rich);
+
+        sender = caver.wallet.keyring.generate();
+        fillKlay(sender, "200");
+        feePayer = caver.wallet.keyring.generate();
+        fillKlay(feePayer, "200");
+    }
+
+    // rich sends enough KLAY to an account.
+    private void fillKlay(AbstractKeyring keyring, String amount) throws IOException {
+        caver.wallet.add(keyring);
+        ValueTransfer valueTransfer = caver.transaction.valueTransfer.create(
+                TxPropertyBuilder.valueTransfer()
+                        .setFrom(rich.getAddress())
+                        .setTo(sender.getAddress())
+                        .setValue(caver.utils.convertToPeb(amount, Utils.KlayUnit.KLAY))
+                        .setGas(BigInteger.valueOf(6000000))
+        );
+        caver.wallet.sign(rich.getAddress(), valueTransfer);
+        caver.rpc.klay.sendRawTransaction(valueTransfer).send();
+    }
+
+    private void generateTxsBomb(int num) throws IOException {
+        if (num == 0) {
+            num = 100;
+        }
```

Comment:
```suggestion
    private void generateTxsBomb() throws IOException {
        generateTxsBomb(100)
    }
    private void generateTxsBomb(int num) throws IOException {
```
How about changing like this?

Reply:
Thanks for your feedback :) 
I added it!